pip/commands/hash.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import absolute_import
+
+import hashlib
+import logging
+import sys
+
+from pip.basecommand import Command
+from pip.cmdoptions import strong_hashes
+from pip.exceptions import FAVORITE_HASH
+from pip.status_codes import ERROR
+from pip.utils import read_chunks
+
+
+logger = logging.getLogger(__name__)
+
+
+class HashCommand(Command):
+    """
+    Compute a hash of a local package archive.
+
+    These can be used with --hash in a requirements file to do repeatable
+    installs.
+
+    """
+    name = 'hash'
+    usage = """%prog [options] <file> ..."""
```

Comment:
Might it be worth keeping then in case this usage string becomes multi-line as well?